internal/splunkconfig/config/capabilityname_test.go

```diff
@@ -20,10 +20,8 @@ func TestCapability_validate(t *testing.T) {
 	tests := validatorTestCases{
 		// empty name is invalid
 		{CapabilityName(""), true},
-		// symbols (including dashes) are invalid
-		{CapabilityName("-"), true},
 		// valid name contains lowercase, uppercase, number, underscore
```

Comment:
Should we add dash to this as well?
```suggestion
		// valid name contains lowercase, uppercase, number, underscore, dash
```